전예찬/layouts/TodoList.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="ko">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../css/temp.css">
+        <script src="../assets/js/test.js"></script>
+        <title>To-Do List</title>
+    </head>
+
+    <body>
+        <div class="container">
+            <h1>2025. 04. 27.</h1>
+            <h2>일요일</h2>
+            <p class="remain">할 일 <span id="remain-cnt">1</span>개 남음</p>
+            
+            <hr>
+
+            <div class="todo">
+                <input type="checkbox">
+                <label>아침 산책</label>
```

Comment:
```suggestion
                <input type="checkbox" id="todo1">
                <label for="todo1">아침 산책</label>
```
이런 식으로 id와 for 값을 주면 관리하기 더 용이해요!